reo/validators.py

```diff
@@ -1107,6 +1140,16 @@ def check_special_cases(self, object_name_path, template_values=None, real_value
                     electric_tariff['urdb_response'] = rate.urdb_dict
                     self.validate_urdb_response()
 
+            if electric_tariff.get('urdb_response') is not None and len(self.urdb_errors)==0:
+                if len(electric_tariff['urdb_response'].get('energyweekdayschedule',[[]])[0]) > 24:
+                    energy_rate_resolution = int(len(electric_tariff['urdb_response'].get('energyweekdayschedule',[[]])[0]) / 24)
```

Comment:
Does the URDB provide schedules that are not hourly? Could you please add a comment or two describing what's happening here? Like `if ...  # then the URDB rate is ...`